pkg/tasks/complexImageTask.go

```diff
@@ -0,0 +1,118 @@
+package tasks
```

Comment:
split this file into 2 files: types related to recaptcha and types related to hcaptcha

Reply:
done